.github/dependabot.yaml

```diff
@@ -0,0 +1,39 @@
+version: 2
```

Comment:
can this file be added to the parent ticket so people know we need to do it.

we also need to backfill the repo's already pinned